src/controllers/user/createUser.ts

```diff
@@ -69,7 +81,7 @@ export const createUser = async (req: Request, res: Response) => {
 
   } catch (err) {
     console.log("Error creating user:", err.message)
-    if (err.message .substring(0,46) == "duplicate key value violates unique constraint") {
+    if (err.message.substring(0,46) == "duplicate key value violates unique constraint") {
```

Comment:
I dont think you should be checking for error conditions using the error message. See if it exists in the db instead

Reply:
So I asked Soumitra this on the server and he said this is how we should do it.

Reply:
Ok this one's fixed. Review again.